rust/processor/migration/src/m20220101_000001_create_table.rs

```diff
@@ -0,0 +1,155 @@
+use sea_orm::Statement;
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
```

Comment:
I noticed `transaction_version` is used as the first part of primary key for many tables, won't it cause the bottleneck on write qps since we are roughly writing our data in monotonic increasing order and most recent writes will hit the same machine and prevent us from horizontal scaling the write qps?